sda-validator/orchestrator/validators/validators_test.go

```diff
@@ -0,0 +1,127 @@
+package validators
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+)
+
+type ValidatorsTestSuite struct {
+	suite.Suite
+
+	tempDir string
+
+	mockCommandExecutor *mockCommandExecutor
+}
+
+func (ts *ValidatorsTestSuite) SetupTest() {
```

Comment:
Since there is `Init()` in `validators.go`, `Init()` will be called every time when a test function is run, and therefore, test may pollute each other. Better to reset the global states for validators at the SetupTest(), e.g. `	Validators = make(map[string]*ValidatorDescription)`